src/connection-monitor-preview/azext_connection_monitor_preview/commands.py

```diff
@@ -0,0 +1,75 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.commands import CliCommandType
+
+from ._client_factory import cf_nw_connection_monitor
+from .profiles import CUSTOM_NW_CONNECTION_MONITOR
+from ._validators import (get_network_watcher_from_location,
+                          process_nw_cm_create_namespace,
+                          process_nw_cm_v2_endpoint_namespace,
+                          process_nw_cm_v2_output_namespace)
+
+
+def load_command_table(self, _):
+
+    nw_connection_monitor_sdk = CliCommandType(
+        operations_tmpl='azext_connection_monitor_preview.vendored_sdks.v2019_11_01.'
+                        'operations#ConnectionMonitorsOperations.{}',
+        client_factory=cf_nw_connection_monitor,
+        resource_type=CUSTOM_NW_CONNECTION_MONITOR,
+        min_api='2019-11-01'
+    )
+
+    with self.command_group('network watcher connection-monitor',
+                            nw_connection_monitor_sdk,
+                            client_factory=cf_nw_connection_monitor,
+                            min_api='2018-01-01') as g:
+        g.custom_command('create', 'create_nw_connection_monitor', validator=process_nw_cm_create_namespace)
```

Comment:
could we add more help information in `_help.py` for this command? like what we need to create a V2 cm. I mean in long summary part.

Reply:
Could you be more specifically?

Reply:
Long summary here:
![image](https://user-images.githubusercontent.com/14357159/73659655-a810bd80-46d1-11ea-87a7-741dca004d0a.png)